woocommerce/woocommerce
# Revert "Add missing Ukrainian Subdivisions to States.php (#35493)"

This reverts commit 076f6f28390decadff5f329a4134cec44e45db1a.

This is a revert of https://github.com/woocommerce/woocommerce/pull/35493

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .






### How to test the changes in this Pull Request:



1. Compare against the original PR
2. See API tests passing to confirm the merge conflict in the api tests were correctly resolved



### Other information:




### FOR PR REVIEWER ONLY:
